### PR TITLE
Fetch cashback from PCMS with locale from member-service

### PR DESCRIPTION
**Pull request by joacimastrom**
# Jira Issue: [IPC-390]

## What?
- Get selected cashbackOptionId and locale from member-service
- Fetch selectedCashback from PCMS
- Fetch cashbackOptions from PCMS

## Why?
- Killing content-service

## Optional checklist
- [ ] Unit tests written
- [X] Tested locally
- [X] Codescouted

[IPC-390]: https://hedvig.atlassian.net/browse/IPC-390?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/resolvers/uncategorised/cashback.ts [hunk 1]
```diff
@@ -1,33 +1,26 @@
...
+    locale: pickedLocale,
+  } = await getSelectedCashbackOptionId(token, headersAndLocale)
 
-export const cashbackInner = async (
-  token: string,
-  headers: ForwardHeaders,
-): Promise<Maybe<Cashback>> => {
-  const options = await getCashbackOptions(token, headers)
+  console.log('locale', locale)
+  console.log('pickedLocale', pickedLocale)
```
Comment (rasmusguterstam) [inline src/resolvers/uncategorised/cashback.ts:21]: Is this on purpose?

**Review by rasmusguterstam — APPROVED** [review]
🤝

File: src/resolvers/uncategorised/cashback.ts (hunk 1 above)
Comment (joacimastrom) [reply src/resolvers/uncategorised/cashback.ts:21]: Very much not so... 😅

File: src/api/upstreams/promiseCms.ts [hunk 2]
```diff
@@ -125,6 +132,21 @@ export const createPromiseCmsClient = (
...
   },
+  getCashbackOptions: async (locale) => {
+    const res = await client.get(`/public/cashback-options/${locale}`, {
+      noAuthorization: true,
+    })
+    return (await res.json()) || []
+  },
+  getCashbackOption: async (cashbackOptionId, locale) => {
+    const res = await client.get(
+      `/public/cashback-option/${cashbackOptionId}/${locale}`,
```
Comment (cpiehl1) [inline src/api/upstreams/promiseCms.ts:143]: `cashback-option` here but `cashback-options` above. Is this to fetch multiple vs one?

**Review by AlexSJ1990 — APPROVED** [review]
✅

File: src/api/upstreams/promiseCms.ts (hunk 2 above)
Comment (cpiehl1) [reply src/api/upstreams/promiseCms.ts:143]: Maybe a more RESTy pattern would be to do `cashback-options` for multiple and `cashback-options/{id}` for one, with the locale being a request parameter. But I guess PCMS mostly uses locale as a path variable atm so maybe it is just better to follow that pattern.

Comment (joacimastrom) [reply src/api/upstreams/promiseCms.ts:143]: yeah I agree, lets change it to plural always

Comment (joacimastrom) [reply src/api/upstreams/promiseCms.ts:143]: I think all endpoints for collections should be pluralized, so I'll change it :)

**Pull request merged**